.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
     - TOX_ENV=py27
     - TOX_ENV=py34
     - TOX_ENV=py35
+    - TOX_ENV=py36
```

Comment:
Probably needs a change on line 4, and using the travix matrix system to map travis versions to tox envs.